VoxelSniperCore/src/test/java/com/github/kevindagame/voxelsniper/AbsTest.java

```diff
@@ -0,0 +1,30 @@
+package com.github.kevindagame.voxelsniper;
+
+import org.junit.Test;
+
+public class AbsTest {
```

Comment:
what was the point of this? I don't see any assert or anything

Reply:
This was to figure out some logic, no longer needed